SKILL.tsv

```diff
@@ -1374,7 +1374,7 @@ SKILL_20150317_001373	{memo X}Attack #{SkillAtkAdd}#{nl}Duration 10 seconds
 SKILL_20150317_001374	$Cloaking
 SKILL_20150317_001375	{memo X}Hide from enemy.
 SKILL_20150317_001376	$Duration: #{CaptionTime}# seconds
-SKILL_20150317_001377	$Undistance
+SKILL_20150317_001377	$Perspective Distortion
```

Comment:
Range Distortion would sound better since this is a Scout move.

'Perspective' still feels too vague.